modules/rosa-aws-provisioned.adoc

```diff
@@ -57,16 +57,22 @@ Two buckets are required with a typical size of 2TB each.
 
 [id="rosa-vpc_{context}"]
 == VPC
-Customers should expect to see one VPC per cluster. Additionally, the VPC will need the following configurations:
+Customers should expect to see one VPC per cluster. Additionally, the VPC will need the following configurations.
 
-* *Subnets*: Two subnets for a cluster with a single availability zone, or six subnets for a cluster with multiple availability zones.
+=== Without AWS PrivateLink
 
+* *Subnets*: Two subnets for a cluster with a single availability zone, or six subnets for a cluster with multiple availability zones.
```

Comment:
There should be some clarification for `six subnets for a cluster with multiple availability zones.` that two subnets per zone